src/apps/reservation-list/list/reservation-list.test.ts

```diff
@@ -927,6 +929,7 @@ describe("Reservation list", () => {
             pickupDeadline: null,
             expiryDate: "2022-09-21",
             dateOfReservation: "2022-06-14T09:00:50.059",
+            //  identifier: "9781844135417", //
```

Comment:
Please do not add commented out code in VCS unless you provide a comment explaining why it is relevant.